x-pack/filebeat/module/salesforce/setupaudittrail/ingest/pipeline.yml

```diff
@@ -0,0 +1,185 @@
+---
+description: Pipeline for parsing Salesforce SetupAuditTrail logs
+processors:
+    - json:
+          field: message
+          target_field: json
+          on_failure:
+              - drop: {}
+
+    - set:
+          field: event.ingested
+          copy_from: _ingest.timestamp
+    - rename:

```

Comment:
Some metrisets are using renaming of message and some are using set.
Can we have unanimity here?

Reply:
Thanks.

Although I do not see any rename, all are using `set` but yes one was getting set with a templated snippet and one using copy_from. Fixed it.

Reply:
This very place where I have mentioned the comment is using rename of message and other filesets are using set

Reply:
Okay, you are talking about `event.original`. But if you see the context selected here, it is for event.ingested: 10-12 lines.

Reply:
Done, thanks!